app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Recipe Details</h1>
+
+<p><b>Name: </b><%= @recipe.name %></p>
+<p><b>Complexity: </b><%= @recipe.complexity %></p>
+<p><b>Genre: </b><%= @recipe.genre %></p>
+<h2>Ingredients:</h2>
+  <% @ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
```

Comment:
good job with this view; if you use `<li>` elements they should be enclosed in a `<ul>` for bullet points, or `<ol>` for numbered lists. Ex: 
```
<ul> 
  <% @ingredients.each do |ingredient| %>
    <li>...</li>
  <% end>
</ul>
```